File: Web/EvaMittal_EvaMittal_2125cs1187_2/GettingStarted/PrintAllPrimesTillN.java
import java.util.*;
public class PrintAllPrimesTillN {
    public static void main(String[] args) {
    Scanner sc= new Scanner(System.in);
    long low = sc.nextLong();
    long high = sc.nextLong();
    long i,j;
    int f;
    for(i=low;i<=high;i++)
    { 
        if(i<=1)
        continue;

        f=0;
        for (j=2;j*j<=i;j++)
        {
            if(i%j==0)
            {
                f=1;
                break;
            }
        }
        if(f==0)
        System.out.println(i);
    }
    }
}

File: Web/Vaishnavi-Singh20002_VaishnaviSingh_2125ec1060_02/GettingStarted/printallprimestilln.cpp
#include <iostream>
using namespace std;
int main(){
    int low, high;
   
    cin >> low >> high;
    for(int i=low;i<=high ; i++)
    { 
           int count=0;
        for(int j=2; j*j<=i;j++)
        {
            if(i%j==0)
            {
                count++;
                break;
            }
        }
        if(count==0)
        cout<<i<<"\n";
    
    }
    
    return 0;

    

}